poi-ooxml/src/main/java/org/apache/poi/ooxml/POIXMLDocumentPart.java

```diff
@@ -18,12 +18,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 
 import java.io.IOException;
 import java.net.URI;
-import java.util.ArrayList;
```

Comment:
can you revert this change to this imports?

Reply:
Done